app/views/gem_categories/_category.html.slim

```diff
@@ -1,5 +1,9 @@
 div id="category-#{category.id}" class="category #{'hidden' if position > 10}"
-  = link_to "#{category.name}", gem_category_path(category), class: 'parental-category-name'
+  div
+    = link_to category.name, gem_category_path(category), class: 'parental-category-name'
+    '
```

Comment:
uvozovka je vicemene ekvivalentni `&nbsp;` na konci, takze kdyz mas treba

```
p
   ' prd
   | brd
```

tak to bude jako `<p>prd brd</p>` a ne `<p>prdbrd</p>`.